db/models.py

```diff
@@ -1 +1,43 @@
 from django.db import models
+
+
+class Race(models.Model):
+    race_to_choose = [
+        ["Elf", "Elf"],
+        ["Dwarf", "Dwarf"],
+        ["Human", "Human"],
+        ["Ork", "Ork"]
+    ]
+    name = models.CharField(
+        max_length=255,
+        unique=True,
+        choices=race_to_choose
+    )
+    description = models.TextField(blank=True)
```

Comment:
Consider using `null=True` for the `description` field to be consistent with the `description` field in the `Guild` model.